examples/karaf-rest-example/README.md

```diff
@@ -0,0 +1,47 @@
+# Apache Karaf REST Example
+
+## Abstract
+
+This example shows how to use JAX-RS to implement a REST service. The entity manager is used in the persistence implementation.
```

Comment:
Maybe we should mention that we use Apache CXF for the REST impl. I would also add an example later using Aries JAXRS whiteboard pattern (that actually use CXF as well).

Reply:
Alright, I will update adding the use or Apache CXF impl. And OK for the Aries JAXRS whiteboard pattern, I check this.